--- origsrc/libf2c/makefile.u	2007-04-25 00:48:46.359375000 -0500
+++ src/libf2c/makefile.u	2007-04-25 01:16:44.343750000 -0500
@@ -84,7 +84,7 @@
 ## of "cc -shared".
 
 libf2c.so: $(OFILES)
-	cc -shared -o libf2c.so $(OFILES)
+	cc -shared -Wl,--enable-auto-image-base -Wl,--out-implib,libf2c.dll.a -o cygf2c.dll $(OFILES) -lfrtbegin
 
 ### If your system lacks ranlib, you don't need it; see README.
 
@@ -183,8 +183,8 @@
 arith.h: arithchk.c
 	$(CC) $(CFLAGS) -DNO_FPINIT arithchk.c -lm ||\
 	 $(CC) -DNO_LONG_LONG $(CFLAGS) -DNO_FPINIT arithchk.c -lm
-	./a.out >arith.h
-	rm -f a.out arithchk.o
+	./a.exe >arith.h
+	rm -f a.exe arithchk.o
 
 check:
 	xsum Notice README abort_.c arithchk.c backspac.c c_abs.c c_cos.c \
